.github/workflows/build.yml

```diff
@@ -109,8 +114,9 @@ jobs:
       - name: cargo test (debug; rustls-provider-test; all features)
         run: cargo test --all-features -p rustls-provider-test
 
-      - name: cargo package --all-features -p portable-rustls
-        run: cargo package --all-features -p portable-rustls
+      # XXX TBD IMPROVE LABEL - ???
+      - name: cargo package ${{ env.ALL_FEATURES_EXCEPT_UNSAFE }} -p portable-rustls
+        run: cargo package ${{ env.ALL_FEATURES_EXCEPT_UNSAFE }} -p portable-rustls
```

Comment:
I think it would be ideal to improve the label for this step in the upstream RUSTLS crate - I would like to propose this when I get a chance.